### PR TITLE
Update README to reflect new care for API stability.

### PR DESCRIPTION
**Pull request by Bluebugs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,16 @@ func NewProgressCallback(win fyne.Window)
 
 func NewExitCallback(app fyne.App, win fyne.Window)
 ```
+
+## API Compatibility Promises
+The master branch of `fyneselfupdate` is *not* guaranteed to have a stable API over time. Still we will try hard to not break its API unecessarily and will follow a proper versionning of our release when necessary. We will also keep it in sync and up to date with `fynelabs/selfupate`.
```
Comment (andydotxyz) [inline README.md]: The branch is `main`

File: README.md [hunk 2]
```diff
@@ -42,3 +42,16 @@ func NewProgressCallback(win fyne.Window)
...
+
+The `fyneselfupdate` package makes the following promises about API compatibility:
+1. A list of all API-breaking changes will be documented in this README.
+1. `fyneselfupdate` will strive for as few API-breaking changes as possible.
+
+## API Breaking Changes
```
Comment (andydotxyz) [inline README.md]: I'm not sure if we need it for this repo, as there has been no breaking changes...

**Review by andydotxyz — CHANGES_REQUESTED** [review]
branch name :)

File: README.md (hunk 2 above)
Comment (Bluebugs) [reply README.md]: I guess we can add it later if necessary.

File: README.md [hunk 3]
```diff
@@ -44,14 +44,11 @@ func NewExitCallback(app fyne.App, win fyne.Window)
 ```
 
 ## API Compatibility Promises
-The master branch of `fyneselfupdate` is *not* guaranteed to have a stable API over time. Still we will try hard to not break its API unecessarily and will follow a proper versionning of our release when necessary. We will also keep it in sync and up to date with `fynelabs/selfupate`.
+The main branch of `fyneselfupdate` is *not* guaranteed to have a stable API over time. Still we will try hard to not break its API unecessarily and will follow a proper versionning of our release. We will also keep it in sync and up to date with `fynelabs/selfupate`.
```
Comment (andydotxyz) [inline README.md]: Ouch, sorry - I just noticed "versionning" typo.

**Review by andydotxyz — CHANGES_REQUESTED** [review]

**Review by andydotxyz — APPROVED** [review]

**Pull request merged**